constant/events.go

```diff
@@ -62,7 +62,10 @@ const (
 	EventReasonReady    = "Ready"
 	EventReasonUploaded = "Uploaded"
 
-	EventReasonUpgrade = "Upgrade"
+	EventReasonUpgrade                = "Upgrade"
+	EventReasonFailedUpgradePreCheck  = "FailedUpgradePreCheck"
+	EventReasonFailedUpgradePostCheck = "FailedUpgradePostCheck"
+	EventReasonPassedUpgradeCheck     = "UpgradeCheckPassed"
```

Comment:
How about "`UpgradePreCheckFailed` and `UpgradeCheckPassed`" or "`FailedUpgradePreCheck` and `PassedUpgradeCheck`"

Reply:
I'll make it more consistent.